bundle/Resources/config/services/controller.yaml

```diff
@@ -0,0 +1,133 @@
+services:
+    _defaults:
+        autoconfigure: true
```

Comment:
I would rather you add the required tag to all services then enable autoconfiguration. IMO, autoconfiguration and autowiring should only be used in project code and not in vendor libs. Even symfony recommends so. https://symfony.com/doc/current/service_container/autowiring.html#public-and-reusable-bundles

Reply:
That recommendation is about autowiring, not autoconfiguration. But I would actually agree, this felt OK since it's contained in one file. I'll update.

Reply:
> That recommendation is about autowiring, not autoconfiguration.

Sure, but I kinda assumed the same goes for autoconfiguration. Better to be explicit in both cases.

Reply:
Updated in https://github.com/netgen/site-bundle/pull/39/commits/ef497bd68e2f549e54d46fca4fadc5b010427574